<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewReconstructionDialog</class>
 <widget class="QDialog" name="NewReconstructionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>298</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>Reconstruction ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="idLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="programIDLabel">
       <property name="text">
        <string>Program ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="programIDComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="internalExecutionCheckBox">
     <property name="text">
      <string>Internal execution</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="hasPrefixesCheckBox">
     <property name="text">
      <string>Has prefixes</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>NewReconstructionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>NewReconstructionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
